docs/learn-transaction-fees.md

```diff
@@ -0,0 +1,81 @@
+
+Web3 Foundation designed the Polkadot fee system with the following objectives:
+
+- Each Relay Chain block should be processed efficiently to avoid delays in block production.
+- The growth rate of the Relay Chain should be bounded.
+- Each block should have space for special, high-priority transactions like misconduct reports.
+- The system should be able to handle spikes in demand.
+- Fees should change slowly so that senders can accurately predict the fee for a given transaction.
+
+## Fee Calculation
+
+Fees on the Polkadot Relay Chain are calculated based on four parameters:
+
+- A base fee
+- A per-byte fee (also known as the "length fee")
+- A weight fee
+- A tip (optional)
+
+The base fee is a fixed fee applied to every transaction. The length fee is simply the product of a constant per-byte fee and the size of the transaction in bytes.
+
+Weights are simply a fixed number designed to represent overall resource consumption of a transaction. Resources could be CPU cycles, memory usage, disk I/O, etc. Each transaction has a weight, which is multiplied by a per-weight fee to calculate the transaction's weight fee.
+
+Tips are an optional transaction fee that users can add to give a transaction higher priority.
+
+Together, these four fees constitute the inclusion fee. This fee is deducted from the sender's account prior to transaction execution. A portion of the fee will go to the block producer and the remainder will go to the [Treasury](learn-treasury). At genesis, this is set to 20% and 80%, respectively.
+
+## Block Limits and Transaction Priority
+
+Blocks in Polkadot have both a maximum length (in bytes) and a maximum weight. Block producers will fill blocks with transactions up to these limits. A portion of each block - currently 25% - is reserved for critical transactions that are related to the chain's operation, such as misbehavior reports. Block producers will only fill up to 75% of a block with normal transactions.
+
+Block producers prioritize transactions based on each transaction's total fee. Since a portion of the fee will go to the block producer, producers will include the transactions with the highest fees to maximize their reward.
+
+## Fee Adjustment
+
+Transaction volume on blockchains is highly irregular, and therefore transaction fees need a mechanism to adjust. However, users should be able to predict transaction fees.
+
+Polkadot uses a slow-adjusting fee mechanism with tips to balance these two considerations. In addition to block _limits,_ Polkadot also has a block fullness _target._ Fees increase or decrease for the next block based on the fullness of the current block relative to the target. The per-weight fee can change up to 30% in a 24 hour period. This rate captures long-term trends in demand, but not short-term spikes. To consider short term spikes, Polkadot uses tips on top of the base, byte, and weight fees. Users can optionally add a tip to the fee to give the transaction a higher priority.
```

Comment:
This is incredibly useful, didn't know this. Where can one find out more about the target? Is it also subject to governance?

Reply:
Hmm, our docs are a bit light. Here are a few:
- Fees Page: https://substrate.dev/docs/en/next/development/module/fees#fee-multiplier
- Research: https://research.web3.foundation/en/latest/polkadot/Token%20Economics.html#-2.-slow-adjusting-mechanism
- Sub 0: https://www.youtube.com/watch?v=mN0ueyTWr_8

In Substrate, `NextFeeMultiplier` is abstract and is made concrete in the runtime (so it's up to the runtime to decide what this multiplier should be).